docs/Getting-Started.md

```diff
@@ -82,11 +82,11 @@ eight elements: the `x` and `z` components of the agent cube's rotation and the
 
 #### Behavior Parameters : Vector Action Space
 
-An Agent is given instructions in the form of a float array of _actions_.
+An Agent is given instructions in the form `ActionBuffers`.
```

Comment:
`ActionBuffers` might be too advanced for the Getting Started section. Should we just drop this sentence?